### PR TITLE
[Storybook Refactor/ Regular Ol' Refactor] Course Blocks

### PR DESCRIPTION
**Pull request by epeach**
While working on the StoryBook refactor, I came across a handfull of files that were essentially doing the same thing - a set of CourseBlocks with optional title/description. To make things a little DRY-er, I replaced these files with one reusable component called "CourseBlocksWrapper". This component takes an optional title, description, link, and set of cards to display.

**List of components replaced with CourseBlocksWrapper:**
``CourseBlocksGradeBands.jsx, CourseBlocksInternationlGradeBands.jsx, CourseBlocksStudentsGradeBands.jsx, CourseBlocksTeacherGradeBands.jsx, CourseBlocksTools.jsx``

**List of stories replaced with CourseBlocksWrapper.story:**
``CourseBlocksInternationlGradeBands.story., CourseBlocksStudentsGradeBands.story., CourseBlocksTeacherGradeBands.story., CourseBlocksTools.story.``

To streamline these components, I also moved all the metadata for the cards to a single constants file: ``CourseBlocksCardsConstants.js``

This change does not introduce any visual changes. Here is a screenshot of a before and after:
Screenshot from 2022-11-17 10-20-08
Screenshot from 2022-11-17 10-19-57

### CODE REVIEW

File: apps/src/util/CourseBlockCardsConstants.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import i18n from '@cdo/locale';
```
Comment (bethanyaconnor) [inline apps/src/util/CourseBlockCardsConstants.js]: nit: it seems like normally our non-react files begin with a lowercase letter. Would it make sense to follow that convention here? (slash is that actually a convention?)

File: apps/src/templates/certificates/StudentsBeyondHoc.jsx [hunk 2]
```diff
@@ -152,8 +153,8 @@ class StudentsBeyondHoc extends Component {
         <h1 style={headingStyle}>{heading}</h1>
         <VerticalImageResourceCardRow cards={cards} />
         {isEnglish && (
-          <CourseBlocksStudentGradeBands
-            showContainer={false}
```
Comment (bethanyaconnor) [inline apps/src/templates/certificates/StudentsBeyondHoc.jsx:156]: what happened to this prop?

Comment (bethanyaconnor) [reply apps/src/templates/certificates/StudentsBeyondHoc.jsx:156]: oh nevermind I see -- you pulled that logic out into its own props that aren't needed here 👍

**Review by bethanyaconnor — APPROVED** [review]
Yay on DRYing up code and net negative diffs ➖

File: apps/src/templates/certificates/StudentsBeyondHoc.jsx (hunk 2 above)
Comment (epeach) [reply apps/src/templates/certificates/StudentsBeyondHoc.jsx:156]: Yup - it basically controlled whether or not text was displayed, so now we can either send that text (heading, descriptions, etc.) through in the `CourseBlocksWrapper` props or, like in this case, just send nothing.

File: apps/src/util/CourseBlockCardsConstants.js (hunk 1 above)
Comment (epeach) [reply apps/src/util/CourseBlockCardsConstants.js]: Good catch - changed!

Comment (davidsbailey) [reply apps/src/util/CourseBlockCardsConstants.js]: idea: a plugin like this one to enforce filename convention: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/filename-case.md

**Pull request merged**